questions/339/explanation.md

```diff
@@ -1,28 +1,39 @@
-You can only `get` the value from a `future` once. It is undefined behaviour to call `get` multiple times,  but implementations are encouraged to throw an exception, and most (such as GCC, Clang and MSVC) also [do so](https://www.godbolt.org/z/K83Ensvnq).
+You can only `get` the value from a `future` once. It is undefined behaviour to call `get` multiple times, but implementations are encouraged to throw an exception, and most (such as GCC, Clang and MSVC) also [do so](https://www.godbolt.org/z/rEWfd7PTd).

```

Comment:
I suggest using the latest fixed version of compilers instead of latest when talking about specific unspecified/undefined/implementation-defined behaviour, and say "do so at the time of writing". Just to avoid confusion if some of them change it in a later version.

Reply:
Personally, I think it's better to use fixed versions everywhere for the same reason (and  because MSVC doesn't have a trunk version), and I actually [used them in the previous commits](https://github.com/knatten/cppquiz23/compare/bca9dd1482ec8a00c260cbd04aa2625ea5fb57f1..950804d14d54ed239faff0d049ea1ff561d025b8). But then I remembered [this thread](https://github.com/knatten/cppquiz/issues/255#issuecomment-1456722274) and changed it back :). Anyway, should we also use fixed versions in [335](https://www.godbolt.org/z/sjvT7Ye1h)?

Reply:
Yeah it's a bit hard to decide. For the site as a whole I think it's prooobably better to use trunk? Since it's kind of nice to show what the latest behaviour is in the general case, and it typically doesn't change? But when we're intentionally demonstrating a particular non-guaranteed behaviour, use fixed versions?

Reply:
OK, let's stick to that plan and use fixed versions in this one and 335